dashboard/src/components/ApplicationStatus/ApplicationStatus.v2.tsx

```diff
@@ -0,0 +1,217 @@
+  const [workloads, setWorkloads] = useState([] as IWorkload[]);
+  const [totalPods, setTotalPods] = useState(0);
+  const [readyPods, setReadyPods] = useState(0);
+
+  useEffect(() => {
+    watchWorkloads();
+    return function cleanup() {
+      closeWatches();
+    };
+  }, [watchWorkloads, closeWatches]);
+
+  useEffect(() => {
+    let currentTotalPods = 0;
+    let currentReadyPods = 0;
+    let currentWorkloads: IWorkload[] = [];
+    [
+      {
+        workloads: flattenItemList(deployments),
+        readyKey: "status.availableReplicas",
+        totalKey: "spec.replicas",
+        type: "deployment",
+      },
+      {
+        workloads: flattenItemList(statefulsets),
+        readyKey: "status.readyReplicas",
+        totalKey: "spec.replicas",
+        type: "statefulset",
+      },
+      {
+        workloads: flattenItemList(daemonsets),
+        readyKey: "status.numberReady",
+        totalKey: "status.currentNumberScheduled",
+        type: "daemonset",
+      },
+    ].forEach(src => {
+      src.workloads.forEach(w => {
+        const wReady = get(w, src.readyKey, 0);
+        const wTotal = get(w, src.totalKey, 0);
+        if (wReady) {
+          currentReadyPods += wReady;
+        }
+        if (wTotal) {
+          currentTotalPods += wTotal;
+        }
+        currentWorkloads = currentWorkloads.concat({
+          name: w.metadata.name,
+          replicas: wTotal,
+          readyReplicas: wReady,
+          type: src.type,
+        });
+      });
+    });
+    setWorkloads(currentWorkloads);
+    setReadyPods(currentReadyPods);
+    setTotalPods(currentTotalPods);
+  }, [deployments, statefulsets, daemonsets]);
+
+  const ready = totalPods === readyPods;
+
+  if (isSomeResourceLoading(deployments.concat(statefulsets).concat(daemonsets))) {
+    return (
+      <div className="center">
+        Loading Status...
+        <LoadingWrapper medium={true} />
+      </div>
+    );
+  }
+  if (info && info.deleted) {
+    return (
+      <div className="center">
+        <div className="color-icon-danger">
+          <CdsIcon shape="exclamation-triangle" size="md" solid={true} /> Application Deleted
+        </div>
+      </div>
+    );
+  }
+  if (info && info.status) {
+    // If the status code is different than "Deployed", display that status
+    const helmStatus = codeToString(info.status);
+    if (helmStatus !== "Deployed") {
+      return (
+        <div className="center">
+          <div className="color-icon-warning">
+            <CdsIcon shape="exclamation-triangle" size="md" solid={true} /> Status: {helmStatus}
+          </div>
+        </div>
+      );
+    }
+  }
+  if (totalPods === 0) {
+    return (
+      <div className="center">
+        <div className="color-icon-info">
+          <CdsIcon shape="exclamation-triangle" size="md" solid={true} /> No Workload Found
+        </div>
+      </div>
+    );
+  }
+  return (
+    <div className="application-status-pie-chart">
```

Comment:
I'd change this element to a `section` and add an `aria-label` to notify this content belongs to an specific section:

```suggestion
    <section aria-label="Application status" className="application-status-pie-chart">
```

This change also applies to the other returned `divs`.